playground/nextjs-app-router/components/demo/Swap.tsx

```diff
@@ -94,7 +95,9 @@ function SwapComponent() {
             token={usdcToken}
             type="to"
           />
-          <SwapButton disabled={true} />
+          <SwapButton
+            disabled={ENVARS[ENVIRONMENT.ENVIRONMENT] != 'development'}
```

Comment:
btw, what you are looking is `VERCEL_ENV === 'preview'` which is different from `NEXT_PUBLIC_VERCEL_ENV`.

Reply:
thanks - yep it's `preview` not `development`

however `NEXT_PUBLIC` prefix should work